pkg/R/SQLContext.R

```diff
@@ -102,11 +148,6 @@ table <- function(sqlCtx, tableName) {
 #' @export
 #' @examples
 #'\dontrun{
-#' sc <- sparkR.init()
```

Comment:
I don't think these examples need to be deleted. Its good style to have complete examples for each example that the user can run